apiserver/src/api/drain.rs

```diff
@@ -1,7 +1,9 @@
 use super::{APIServerSettings, ApiserverCommonHeaders};
-use crate::error::{self, Result};
 use models::node::BottlerocketShadowClient;
 
```

Comment:
I wouldn't add a new line here. no newlines in 'use' statements and let rustfmt do its thing.

```suggestion
```